pkg/alicloud/client/client.go

```diff
@@ -355,3 +358,78 @@ func (c *vpcClient) DescribeNatGateways(req *vpc.DescribeNatGatewaysRequest) (*v
 func (c *vpcClient) DescribeEipAddresses(req *vpc.DescribeEipAddressesRequest) (*vpc.DescribeEipAddressesResponse, error) {
 	return c.client.DescribeEipAddresses(req)
 }
+
+// vpcClient defines the struct of VPC client
+type ramClient struct {
+	client *ram.Client
+}
+
+// NewSTSClient creates a new STS client with given region, AccessKeyID, and AccessKeySecret
+func (f *clientFactory) NewRAMClient(region, accessKeyID, accessKeySecret string) (RAM, error) {
+	client, err := ram.NewClientWithAccessKey(region, accessKeyID, accessKeySecret)
+	if err != nil {
+		return nil, err
+	}
+
+	return &ramClient{
+		client: client,
+	}, nil
+}
+
+func (c *ramClient) GetServiceLinkedRole(regionID, roleName string) (*ram.Role, error) {
```

Comment:
To my knowlege, ServiceLinkedRole is not regional based. Can we remove `regionID`?

Reply:
make sense